pkg/manifest/manifest.go

```diff
@@ -54,24 +42,22 @@ type OAuth struct {
 func (o OAuth) GetTokenEndpointValue() string {
 	if o.TokenEndpoint == nil {
 		return ""
+	} else {
```

Comment:
nit: why introduce the `else` branch here?
seems especially strange as this PR also drops an else from [download_configs.go#getApisToDownload](https://github.com/Dynatrace/dynatrace-configuration-as-code/pull/926/files#diff-3f23ca622bf717266f5e6efcbe7a9c319ffdba2f19a76ed66d1893676d8e5e83R313)

maybe once change is by @jskelin and one by @warber though? How want to deal with else vs early-exit, might a good thing to discuss for coding standards :sweat_smile:

Reply:
on places like this usually I would use elvis operator (?:)
I'll return it back